dist/src/main/java/io/camunda/webapps/controllers/IndexController.java

```diff
@@ -5,22 +5,40 @@
  * Licensed under the Camunda License 1.0. You may not use this file
  * except in compliance with the Camunda License 1.0.
  */
-package io.camunda.operate.webapp;
+package io.camunda.webapps.controllers;
+
+import static io.camunda.application.Profile.*;
 
 import jakarta.servlet.ServletContext;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
 
 @Controller
 public class IndexController {
-
   @Autowired private ServletContext context;
+  @Autowired private Environment environment;
 
   @GetMapping("/index.html")
-  public String index(Model model) {
-    model.addAttribute("contextPath", context.getContextPath() + "/");
-    return "index";
+  public String index() {
+    if (environment.acceptsProfiles(TASKLIST.getId(), "!" + OPERATE.getId())) {
```

Comment:
The same as above, it should be configured what is the default app, so that it redirects to it. It should not know anything about profiles at this place.

Reply:
Done! new property `camunda.webapps.default-app` is introduced and set by `WebappsConfigurationInitializer`